### PR TITLE
CORE-95 - ATC Training Places

### PR DESCRIPTION
**Pull request by AxonC**
Being merged for the purposes of building into Nova. Review requested.

### CODE REVIEW

File: resources/assets/less/admin.less [hunk 1]
```diff
@@ -49,8 +49,7 @@
   overflow: hidden;
 }
 
-.dev_environment_notification {
```
Comment (atoff) [inline resources/assets/less/admin.less]: Meant to be removed?

File: app/Http/Controllers/Adm/Training/WaitingListManagementController.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        $this->authorize('addAccount', $waitingList);
+
+        try {
+            $user = Account::findOrFail($request->get('account_id'));
+        } catch (ModelNotFoundException $e) {
+            return Redirect::route('training.waitingList.show', $waitingList)
+                ->withError('Account Not Found.');
+        }
+
+        $waitingList->addToWaitingList($user, $request->user());
```
[the remark this answers is not in the document]
Comment (AxonC) [reply app/Http/Controllers/Adm/Training/WaitingListManagementController.php]: This logic is handled in the WaitingListAccountRequest FormRequest

REF: 
https://github.com/VATSIM-UK/core/blob/c750cbbb0cb0097cea19c67b94bd78bfc3acf986/app/Http/Requests/Training/WaitingListAccountRequest.php#L32-L42

File: app/Http/Controllers/Adm/Training/WaitingListManagementController.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        $this->authorize('removeAccount', $waitingList);
+
+        try {
+            $user = Account::findOrFail($request->get('account_id'));
+        } catch (ModelNotFoundException $e) {
+            return Redirect::route('training.waitingList.show', $waitingList)
+                ->withError('Account Not Found.');
+        }
+
+        $waitingList->removeFromWaitingList($user);
```
[the remark this answers is not in the document]
Comment (AxonC) [reply app/Http/Controllers/Adm/Training/WaitingListManagementController.php]: See above comment.

File: app/Events/Training/AccountAddedToWaitingList.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     *
+     * @return \Illuminate\Broadcasting\Channel|array
+     */
+    public function broadcastOn()
+    {
+        return new PrivateChannel('channel-name');
```
[the remark this answers is not in the document]
Comment (AxonC) [reply app/Events/Training/AccountAddedToWaitingList.php]: This is only for use with laravel echo.. we aren't using it, no, but I don't see any harm in leaving it in.

File: resources/assets/less/admin.less (hunk 1 above)
Comment (AxonC) [reply resources/assets/less/admin.less]: Changed back in ec64d6c

File: app/Events/Training/AccountAddedToWaitingList.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (atoff) [reply app/Events/Training/AccountAddedToWaitingList.php]: Personally I would either comment the line out until we utilise it, or change `channel-name` to something that we might use in the future. Either way, don't really mind.

**Pull request merged**